// @strapi/admin/build/412.72afdf0c.chunk.js
(self.webpackChunk_strapi_admin=self.webpackChunk_strapi_admin||[]).push([[412],{91131:function($,c,t){var n=t(94318),o=t(3387),s="[object RegExp]";function a(f){return o(f)&&n(f)==s}$.exports=a},63349:function($,c,t){var n=t(5720),o=/^\s+/;function s(a){return a&&a.slice(0,n(a)+1).replace(o,"")}$.exports=s},5720:function($){var c=/\s/;function t(n){for(var o=n.length;o--&&c.test(n.charAt(o)););return o}$.exports=t},52654:function($,c,t){var n=t(91131),o=t(76535),s=t(91782),a=s&&s.isRegExp,f=a?o(a):n;$.exports=f},78746:function($,c,t){var n=t(63349),o=t(85973),s=t(22945),a=0/0,f=/^[-+]0x[0-9a-f]+$/i,e=/^0b[01]+$/i,i=/^0o[0-7]+$/i,h=parseInt;function E(p){if(typeof p=="number")return p;if(s(p))return a;if(o(p)){var _=typeof p.valueOf=="function"?p.valueOf():p;p=o(_)?_+"":_}if(typeof p!="string")return p===0?p:+p;p=n(p);var w=e.test(p);return w||i.test(p)?h(p.slice(2),w?2:8):f.test(p)?a:+p}$.exports=E},51382:function($,c,t){var n=t(90454),o=t(24793),s=t(34327),a=t(85973),f=t(52654),e=t(38788),i=t(51119),h=t(25225),E=t(41119),p=30,_="...",w=/\w*$/;function b(P,m){var u=p,M=_;if(a(m)){var j="separator"in m?m.separator:j;u="length"in m?h(m.length):u,M="omission"in m?n(m.omission):M}P=E(P);var y=P.length;if(s(P)){var R=i(P);y=R.length}if(u>=y)return P;var O=u-e(M);if(O<1)return M;var T=R?o(R,0,O).join(""):P.slice(0,O);if(j===void 0)return T+M;if(R&&(O+=T.length-O),f(j)){if(P.slice(O).search(j)){var L,W=T;for(j.global||(j=RegExp(j.source,E(w.exec(j))+"g")),j.lastIndex=0;L=j.exec(W);)var N=L.index;T=T.slice(0,N===void 0?O:N)}}else if(P.indexOf(n(j),O)!=O){var A=T.lastIndexOf(j);A>-1&&(T=T.slice(0,A))}return T+M}$.exports=b},18101:function($,c,t){"use strict";t.d(c,{Q:function(){return p}});var n=t(74512),o=t(32735),s=t(14911),a=t(8471),f=t(73290),e=t(65351),i=t(84642),h=t(8365);const E=(0,a.ZP)(e.G)`
  &[aria-disabled='true'] {
    ${f.sg}
    &:active {
      ${f.sg}
    }
  }
  &:hover {
    ${f.yP}
  }
  &:active {
    ${f.tB}
  }
  ${f.PD}
`,p=o.forwardRef(({variant:_="default",startIcon:w,endIcon:b,disabled:P=!1,children:m,size:u="S",href:M,to:j,...y},R)=>{const O=M?"_blank":void 0,T=M?"noreferrer noopener":void 0,L=u==="S"?2:"10px",W=4;return(0,n.jsxs)(E,{ref:R,"aria-disabled":P,size:u,variant:_,target:O,rel:T,to:P?void 0:j,href:P?"#":M,background:"buttonPrimary600",borderColor:"buttonPrimary600",hasRadius:!0,gap:2,inline:!0,paddingBottom:L,paddingLeft:W,paddingRight:W,paddingTop:L,pointerEvents:P?"none":void 0,...y,as:j&&!P?s.OL:"a",children:[w&&(0,n.jsx)(i.k,{"aria-hidden":!0,children:w}),(0,n.jsx)(h.Z,{variant:u==="S"?"pi":void 0,fontWeight:"bold",textColor:"buttonNeutral0",children:m}),b&&(0,n.jsx)(i.k,{"aria-hidden":!0,children:b})]})})},63057:function($,c,t){"use strict";t.d(c,{Q:function(){return a}});var n=t(74512),o=t(8471),s=t(74323);const a=({options:e,...i})=>(0,n.jsx)(s.NU,{...i,children:e.map(h=>"children"in h?(0,n.jsx)(s.Ab,{label:h.label,values:h.children.map(E=>E.value.toString()),children:h.children.map(E=>(0,n.jsx)(f,{value:E.value,children:E.label},E.value))},h.label):(0,n.jsx)(s.ML,{value:h.value,children:h.label},h.value))}),f=(0,o.ZP)(s.ML)`
  padding-left: ${({theme:e})=>e.spaces[7]};
`},70788:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M24 13.3a.2.2 0 0 1-.2.2H5.74l8.239 8.239a.2.2 0 0 1 0 .282L12.14 23.86a.2.2 0 0 1-.282 0L.14 12.14a.2.2 0 0 1 0-.282L11.86.14a.2.2 0 0 1 .282 0L13.98 1.98a.2.2 0 0 1 0 .282L5.74 10.5H23.8c.11 0 .2.09.2.2v2.6Z"})}),s=o},90984:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 34 25",...a,children:[(0,n.jsx)("rect",{width:33,height:23,x:.5,y:1,fill:"#EAF5FF",stroke:"#B8E1FF",rx:2.5}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M18.901 9.828a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M19.703 8.785a.81.81 0 0 1-.512.748.814.814 0 0 1-.91-.239.804.804 0 0 1 .753-1.301.814.814 0 0 1 .669.792c.005.311.487.311.483 0a1.308 1.308 0 0 0-.867-1.215 1.288 1.288 0 0 0-1.4.39 1.296 1.296 0 0 0-.119 1.489c.283.468.83.697 1.364.596.597-.113 1.012-.664 1.021-1.258.005-.314-.477-.314-.482-.002ZM18.901 13.488a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M19.703 12.445a.81.81 0 0 1-.512.748.814.814 0 0 1-.91-.239.804.804 0 0 1 .753-1.301.812.812 0 0 1 .669.792c.005.311.487.311.483 0a1.307 1.307 0 0 0-.867-1.215 1.288 1.288 0 0 0-1.4.39 1.296 1.296 0 0 0-.119 1.489c.283.468.83.697 1.364.596.597-.113 1.012-.664 1.021-1.258.005-.314-.477-.314-.482-.002ZM18.901 17.247a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M19.703 16.204a.81.81 0 0 1-.512.748.814.814 0 0 1-.91-.239.804.804 0 0 1 .753-1.301.812.812 0 0 1 .669.792c.005.311.487.311.483 0a1.308 1.308 0 0 0-.867-1.215 1.288 1.288 0 0 0-1.4.39 1.296 1.296 0 0 0-.119 1.489c.283.468.83.698 1.364.596.597-.113 1.012-.664 1.021-1.258.005-.313-.477-.313-.482-.002ZM15.075 9.842a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M15.876 8.8a.81.81 0 0 1-.512.748.814.814 0 0 1-.91-.24.804.804 0 0 1 .753-1.301.81.81 0 0 1 .669.792c.005.312.488.312.483 0a1.308 1.308 0 0 0-.867-1.214 1.288 1.288 0 0 0-1.4.389 1.296 1.296 0 0 0-.119 1.49c.283.468.831.697 1.365.596.596-.114 1.011-.664 1.02-1.258.006-.314-.477-.314-.482-.003ZM15.075 13.503a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M15.876 12.46a.81.81 0 0 1-.512.748.814.814 0 0 1-.91-.24.804.804 0 0 1 .753-1.301.81.81 0 0 1 .669.792c.005.312.488.312.483 0a1.308 1.308 0 0 0-.867-1.214 1.288 1.288 0 0 0-1.4.389 1.296 1.296 0 0 0-.119 1.49c.283.468.831.697 1.365.596.596-.114 1.011-.664 1.02-1.258.006-.314-.477-.314-.482-.003ZM15.075 17.261a1.043 1.043 0 1 0 0-2.086 1.043 1.043 0 0 0 0 2.086Z"}),(0,n.jsx)("path",{fill:"#0C75AF",d:"M15.876 16.218a.81.81 0 0 1-.512.749.814.814 0 0 1-.91-.24.804.804 0 0 1 .753-1.301.808.808 0 0 1 .669.792c.005.312.488.312.483 0a1.308 1.308 0 0 0-.867-1.214 1.288 1.288 0 0 0-1.4.389 1.293 1.293 0 0 0-.119 1.487c.283.468.831.698 1.365.596.596-.113 1.011-.664 1.02-1.258.006-.311-.477-.311-.482 0Z"})]}),s=o},19999:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M13 21v2.5l-3-2-3 2V21h-.5A3.5 3.5 0 0 1 3 17.5V5a3 3 0 0 1 3-3h14a1 1 0 0 1 1 1v17a1 1 0 0 1-1 1h-7Zm-6-2v-2h6v2h6v-3H6.5a1.5 1.5 0 1 0 0 3H7ZM7 5v2h2V5H7Zm0 3v2h2V8H7Zm0 3v2h2v-2H7Z"})}),s=o},23673:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 32 24",...a,children:[(0,n.jsx)("rect",{width:31,height:23,x:.5,y:.5,fill:"#4945FF",stroke:"#4945FF",rx:2.5}),(0,n.jsx)("path",{fill:"#fff",d:"M15.328 10.54h1.723c.012-.089.012-.165.012-.253 0-1.676-1.471-2.959-3.41-2.959-2.696 0-4.647 2.22-4.647 5.344 0 2.15 1.383 3.545 3.504 3.545 2.045 0 3.597-1.154 3.967-2.936h-1.752c-.276.826-1.102 1.371-2.063 1.371-1.137 0-1.846-.802-1.846-2.103 0-2.08 1.19-3.65 2.725-3.65 1.037 0 1.746.62 1.787 1.558v.082ZM21.053 16l1.488-6.943h2.531l.31-1.512H18.54l-.31 1.512h2.53L19.272 16h1.782Z"})]}),s=o},32880:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:[(0,n.jsx)("path",{fill:"#212134",d:"M11.987 23.036v-.964H1.876V1.876h10.111V0H0v24h11.987v-.964Z"}),(0,n.jsx)("path",{fill:"#212134",d:"M8 11.2c0-.11.09-.2.2-.2h11.973l-5.445-5.445a.2.2 0 0 1 0-.283l1.13-1.13a.2.2 0 0 1 .283 0l7.718 7.717a.2.2 0 0 1 0 .282L16.14 19.86a.2.2 0 0 1-.282 0l-1.13-1.13a.2.2 0 0 1 0-.284L20.172 13H8.2a.2.2 0 0 1-.2-.2v-1.6Z"})]}),s=o},18981:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M6.455 19 2 22.5V4a1 1 0 0 1 1-1h18a1 1 0 0 1 1 1v14a1 1 0 0 1-1 1H6.455ZM7 10v2h2v-2H7Zm4 0v2h2v-2h-2Zm4 0v2h2v-2h-2Z"})}),s=o},39544:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M1.946 9.315c-.522-.174-.527-.455.01-.634l19.087-6.362c.53-.176.832.12.684.638l-5.454 19.086c-.15.529-.455.547-.679.045L12 14l6-8-8 6-8.054-2.685Z"})}),s=o},60389:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 32 24",...a,children:[(0,n.jsx)("path",{fill:"#FDF4DC",stroke:"#FAE7B9",d:"M.5 3A2.5 2.5 0 0 1 3 .5h26A2.5 2.5 0 0 1 31.5 3v18a2.5 2.5 0 0 1-2.5 2.5H3A2.5 2.5 0 0 1 .5 21V3Z"}),(0,n.jsx)("path",{fill:"#D9822F",d:"M20.158 11.995c0-.591-.463-1.073-1.045-1.11H13.53V9.245a2.05 2.05 0 0 1 2.046-2.049c1.13 0 2.048.784 2.049 1.913 0 .24.194.433.433.433h.33a.433.433 0 0 0 .433-.433C18.82 7.32 17.365 5.999 15.577 6a3.246 3.246 0 0 0-3.241 3.244v1.642h-.223c-.615 0-1.113.499-1.113 1.114v4.887c.001.615.5 1.113 1.115 1.113l6.93-.003c.616 0 1.114-.5 1.114-1.115l-.001-4.887Z"})]}),s=o},58518:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 8 8",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M2 .93c0-.4.45-.63.78-.41l4.6 3.06c.3.2.3.64 0 .84l-4.6 3.06A.5.5 0 0 1 2 7.07V.93Z"})}),s=o},47372:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 15 14",...a,children:(0,n.jsx)("path",{fill:"#212134",d:"M5.08 4.1c0-1.19 1.18-2.17 2.42-2.17s2.43.98 2.43 2.17c0 1.1-.56 1.61-1.31 2.28l-.03.03c-.75.65-1.66 1.47-1.66 3.09a.57.57 0 1 0 1.15 0c0-1.08.55-1.6 1.3-2.26l.02-.02c.75-.66 1.67-1.48 1.67-3.12C11.07 2.13 9.22.78 7.5.78 5.78.78 3.93 2.13 3.93 4.1a.57.57 0 1 0 1.15 0Zm2.42 9.26a.88.88 0 1 0 0-1.75.88.88 0 0 0 0 1.75Z"})}),s=o},53226:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 32 24",...a,children:[(0,n.jsx)("rect",{width:31,height:23,x:.5,y:.5,fill:"#EAF5FF",stroke:"#B8E1FF",rx:2.5}),(0,n.jsx)("path",{fill:"#0C75AF",fillRule:"evenodd",d:"M19.286 9.286v-.857a.397.397 0 0 0-.138-.302A.465.465 0 0 0 18.82 8h-8.357a.465.465 0 0 0-.326.127.397.397 0 0 0-.138.302v.857c0 .116.046.216.138.301.092.085.2.127.326.127h8.357a.465.465 0 0 0 .327-.127.397.397 0 0 0 .138-.301Zm2.785 2.713v.857a.397.397 0 0 1-.137.301.465.465 0 0 1-.327.128H10.464a.465.465 0 0 1-.326-.128.397.397 0 0 1-.138-.301v-.857c0-.116.046-.217.138-.302a.465.465 0 0 1 .326-.127h11.143c.126 0 .235.043.327.127a.397.397 0 0 1 .137.302Zm-1.857 3.574v.857a.397.397 0 0 1-.137.302.465.465 0 0 1-.327.127h-9.286a.465.465 0 0 1-.326-.127.397.397 0 0 1-.138-.302v-.857c0-.116.046-.216.138-.301a.465.465 0 0 1 .326-.127h9.286c.126 0 .235.042.326.127a.397.397 0 0 1 .138.301Z",clipRule:"evenodd"})]}),s=o},12171:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsxs)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 32 24",...a,children:[(0,n.jsx)("rect",{width:31,height:23,x:.5,y:.5,fill:"#0C75AF",stroke:"#0C75AF",rx:2.5}),(0,n.jsx)("path",{fill:"#fff",d:"M8.523 13.586c.106 1.64 1.418 2.63 3.34 2.63 2.098 0 3.516-1.113 3.516-2.788 0-1.143-.65-1.846-2.086-2.297l-.867-.27c-.797-.252-1.137-.597-1.137-1.066 0-.598.633-1.031 1.459-1.031.873 0 1.512.474 1.617 1.183h1.67c-.053-1.54-1.36-2.619-3.217-2.619-1.91 0-3.328 1.131-3.328 2.678 0 1.09.715 1.922 1.963 2.309l.879.275c.914.287 1.266.592 1.266 1.084 0 .662-.657 1.107-1.606 1.107-.914 0-1.635-.469-1.758-1.195h-1.71ZM20.107 16l1.489-6.943h2.531l.31-1.512h-6.843l-.31 1.512h2.53L18.326 16h1.781Z"})]}),s=o},22356:function($,c,t){"use strict";t.d(c,{Z:function(){return s}});var n=t(74512);const o=a=>(0,n.jsx)("svg",{xmlns:"http://www.w3.org/2000/svg",width:"1rem",height:"1rem",fill:"none",viewBox:"0 0 24 24",...a,children:(0,n.jsx)("path",{fill:"#212134",fillRule:"evenodd",d:"M23.707.297A1 1 0 0 0 23 .004h-2a13.907 13.907 0 0 0-5.38 1.077 1 1 0 0 0-.615.923V4.92a.035.035 0 0 1-.022.038l-2-1.47a1 1 0 0 0-1.265.052A14 14 0 0 0 7 14.004v1.585l-2.707 2.707a1 1 0 1 0 1.415 1.415l2.707-2.708H10a14.014 14.014 0 0 0 14-14v-2a1 1 0 0 0-.293-.706ZM18 23.999H3a3 3 0 0 1-3-3V6A3 3 0 0 1 3 3h3a1 1 0 1 1 0 2H3a1 1 0 0 0-1 1v15a1 1 0 0 0 1 1h15a1 1 0 0 0 1-1v-3a1 1 0 1 1 2 0v3a3 3 0 0 1-3 3Z",clipRule:"evenodd"})}),s=o},83819:function($,c,t){"use strict";t.d(c,{Y:function(){return i}});var n=t(74512),o=t(32735),s=t(8471),a=t(69549),f=t(5717);const e=(0,s.ZP)(f.k)`
  > svg {
    height: ${({theme:h})=>h.spaces[3]};
    width: ${({theme:h})=>h.spaces[3]};

    > g,
    path {
      fill: ${({theme:h})=>h.colors.neutral0};
    }
  }

  &[aria-disabled='true'] {
    pointer-events: none;
  }

  ${a.BF}
`,i=o.forwardRef(({disabled:h,children:E,background:p="neutral0",..._},w)=>(0,n.jsx)(e,{ref:w,"aria-disabled":h,as:"button",type:"button",disabled:h,padding:2,hasRadius:!0,background:p,borderColor:"neutral200",cursor:"pointer",..._,children:E}));i.displayName="BaseButton"},55171:function($,c,t){"use strict";t.d(c,{f:function(){return a}});var n=t(74512),o=t(32735),s=t(16833);const a=o.forwardRef(({href:f,rel:e="noreferrer noopener",target:i="_self",disabled:h=!1,isExternal:E=!1,...p},_)=>(0,n.jsx)(s.x,{as:"a",ref:_,target:E?"_blank":i,rel:E?e:void 0,href:h?"#":f,"aria-disabled":h,cursor:"pointer",...p}));a.displayName="BaseLink"},16833:function($,c,t){"use strict";t.d(c,{x:function(){return f}});var n=t(8471),o=t(40041),s=t(40187);const a={color:!0,cursor:!0,height:!0,width:!0},f=n.ZP.div.withConfig({shouldForwardProp:(e,i)=>!a[e]&&i(e)})`
  // Font
  font-size: ${({fontSize:e,theme:i})=>(0,s.$)(i.fontSizes,e,e)};

  // Colors
  background: ${({theme:e,background:i})=>(0,s.$)(e.colors,i,i)};
  color: ${({theme:e,color:i})=>(0,s.$)(e.colors,i,void 0)};

  // Spaces
  ${({theme:e,padding:i})=>(0,o.Z)("padding",i,e)}
  ${({theme:e,paddingTop:i})=>(0,o.Z)("padding-top",i,e)}
  ${({theme:e,paddingRight:i})=>(0,o.Z)("padding-right",i,e)}
  ${({theme:e,paddingBottom:i})=>(0,o.Z)("padding-bottom",i,e)}
  ${({theme:e,paddingLeft:i})=>(0,o.Z)("padding-left",i,e)}
  ${({theme:e,marginLeft:i})=>(0,o.Z)("margin-left",i,e)}
  ${({theme:e,marginRight:i})=>(0,o.Z)("margin-right",i,e)}
  ${({theme:e,marginTop:i})=>(0,o.Z)("margin-top",i,e)}
  ${({theme:e,marginBottom:i})=>(0,o.Z)("margin-bottom",i,e)}

  // Responsive hiding
  ${({theme:e,hiddenS:i})=>i?`${e.mediaQueries.tablet} { display: none; }`:void 0}
  ${({theme:e,hiddenXS:i})=>i?`${e.mediaQueries.mobile} { display: none; }`:void 0}
  

  // Borders
  border-radius: ${({theme:e,hasRadius:i,borderRadius:h})=>i?e.borderRadius:h};
  border-style: ${({borderStyle:e})=>e};
  border-width: ${({borderWidth:e})=>e};
  border-color: ${({borderColor:e,theme:i})=>(0,s.$)(i.colors,e,void 0)};
  border: ${({theme:e,borderColor:i,borderStyle:h,borderWidth:E})=>{if(i&&!h&&typeof E>"u")return`1px solid ${e.colors[i]}`}};

  // Shadows
  box-shadow: ${({theme:e,shadow:i})=>(0,s.$)(e.shadows,i,void 0)};

  // Handlers
  pointer-events: ${({pointerEvents:e})=>e};
  &:hover {
    ${({_hover:e,theme:i})=>e?e(i):void 0}
  }

  // Display
  display: ${({display:e})=>e};

  // Position
  position: ${({position:e})=>e};
  left: ${({left:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  right: ${({right:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  top: ${({top:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  bottom: ${({bottom:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  z-index: ${({zIndex:e})=>e};
  overflow: ${({overflow:e})=>e};

  // Size
  width: ${({width:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  max-width: ${({maxWidth:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  min-width: ${({minWidth:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  height: ${({height:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  max-height: ${({maxHeight:e,theme:i})=>(0,s.$)(i.spaces,e,e)};
  min-height: ${({minHeight:e,theme:i})=>(0,s.$)(i.spaces,e,e)};

  // Animation
  transition: ${({transition:e})=>e};
  transform: ${({transform:e})=>e};
  animation: ${({animation:e})=>e};

  //Flexbox children props
  flex-shrink: ${({shrink:e})=>e};
  flex-grow: ${({grow:e})=>e};
  flex-basis: ${({basis:e})=>e};
  flex: ${({flex:e})=>e};

  // Text
  text-align: ${({textAlign:e})=>e};
  text-transform: ${({textTransform:e})=>e};
  line-height: ${({theme:e,lineHeight:i})=>(0,s.$)(e.lineHeights,i,i)};

  // Cursor
  cursor: ${({cursor:e})=>e};
`},5717:function($,c,t){"use strict";t.d(c,{k:function(){return f}});var n=t(8471),o=t(40041),s=t(16833);const a={direction:!0},f=(0,n.ZP)(s.x).withConfig({shouldForwardProp:(e,i)=>!a[e]&&i(e)})`
  align-items: ${({alignItems:e="center"})=>e};
  display: ${({display:e="flex",inline:i})=>i?"inline-flex":e};
  flex-direction: ${({direction:e="row"})=>e};
  flex-shrink: ${({shrink:e})=>e};
  flex-wrap: ${({wrap:e})=>e};
  ${({gap:e,theme:i})=>(0,o.Z)("gap",e,i)};
  justify-content: ${({justifyContent:e})=>e};
`},17856:function($,c,t){"use strict";t.d(c,{Z:function(){return P}});var n=t(8471);const o="alpha",s="beta",a="delta",f="epsilon",e="omega",i="pi",h="sigma",E=[o,s,a,f,e,i,h],p=({ellipsis:m=!1})=>m&&`
    display: block;
    white-space: nowrap;
    overflow: hidden;
    text-overflow: ellipsis;
  `,_=({variant:m=e,theme:u})=>{switch(m){case o:return`
        font-weight: ${u.fontWeights.bold};
        font-size: ${u.fontSizes[5]};
        line-height: ${u.lineHeights[2]};
      `;case s:return`
        font-weight: ${u.fontWeights.bold};
        font-size: ${u.fontSizes[4]};
        line-height: ${u.lineHeights[1]};
      `;case a:return`
        font-weight: ${u.fontWeights.semiBold};
        font-size: ${u.fontSizes[3]};
        line-height: ${u.lineHeights[2]};
      `;case f:return`
        font-size: ${u.fontSizes[3]};
        line-height: ${u.lineHeights[6]};
      `;case e:return`
        font-size: ${u.fontSizes[2]};
        line-height: ${u.lineHeights[4]};
      `;case i:return`
        font-size: ${u.fontSizes[1]};
        line-height: ${u.lineHeights[3]};
      `;case h:return`
        font-weight: ${u.fontWeights.bold};
        font-size: ${u.fontSizes[0]};
        line-height: ${u.lineHeights[5]};
        text-transform: uppercase;
      `;default:return`
        font-size: ${u.fontSizes[2]};
      `}};var w=t(40187);const b={fontSize:!0,fontWeight:!0},P=n.ZP.span.withConfig({shouldForwardProp:(m,u)=>!b[m]&&u(m)})`
  ${_}
  ${p}

  // These properties need to come after {variantStyle}, because they might
  // overwrite a variant attribute
  font-weight: ${({theme:m,fontWeight:u})=>(0,w.$)(m.fontWeights,u,void 0)};
  font-size: ${({theme:m,fontSize:u})=>(0,w.$)(m.fontSizes,u,void 0)};
  line-height: ${({theme:m,lineHeight:u})=>(0,w.$)(m.lineHeights,u,u)};
  color: ${({theme:m,textColor:u})=>m.colors[u||"neutral800"]};
  text-align: ${({textAlign:m})=>m};
  text-decoration: ${({textDecoration:m})=>m};
  text-transform: ${({textTransform:m})=>m};
`},40041:function($,c,t){"use strict";t.d(c,{Z:function(){return n}});const n=(o,s,a)=>{if(!s)return;let f=Array.isArray(s)?s:[];if(!Array.isArray(s)&&typeof s=="object"&&(f=[s?.desktop,s?.tablet,s?.mobile]),f.length>0)return f.reduce((h,E,p)=>{if(E)switch(p){case 0:return`${h}${o}: ${a.spaces[E]};`;case 1:return`${h}${a.mediaQueries.tablet}{${o}: ${a.spaces[E]};}`;case 2:return`${h}${a.mediaQueries.mobile}{${o}: ${a.spaces[E]};}`;default:return h}return h},"");const e=a.spaces[s]||s;return`${o}: ${e};`}},40187:function($,c,t){"use strict";t.d(c,{$:function(){return s}});function n(a,f){return typeof a=="string"?!1:f in a}function o(a){return a&&typeof a=="object"&&!Array.isArray(a)}function s(a,f,e){return f&&n(a,f)?a[f]:e}},77062:function($,c,t){"use strict";t.d(c,{M:function(){return a}});var n=t(32735);const o=n["useId".toString()]||(()=>{});let s=0;const a=f=>{const[e,i]=(0,n.useState)(o());return(0,n.useLayoutEffect)(()=>{f||i(h=>h??String(s++))},[f]),f?.toString()??(e||"")}},69549:function($,c,t){"use strict";t.d(c,{BF:function(){return a},k3:function(){return s}});var n=t(8471);const o=f=>({theme:e,size:i})=>e.sizes[f][i],s=(f="&")=>({theme:e,hasError:i=!1})=>(0,n.iv)`
    outline: none;
    box-shadow: 0;
    transition-property: border-color, box-shadow, fill;
    transition-duration: 0.2s;

    ${f}:focus-within {
      border: 1px solid ${i?e.colors.danger600:e.colors.primary600};
      box-shadow: ${i?e.colors.danger600:e.colors.primary600} 0px 0px 0px 2px;
    }
  `,a=({theme:f})=>(0,n.iv)`
  position: relative;
  outline: none;

  &:after {
    transition-property: all;
    transition-duration: 0.2s;
    border-radius: 8px;
    content: '';
    position: absolute;
    top: -4px;
    bottom: -4px;
    left: -4px;
    right: -4px;
    border: 2px solid transparent;
  }

  &:focus-visible {
    outline: none;
    &:after {
      border-radius: 8px;
      content: '';
      position: absolute;
      top: -5px;
      bottom: -5px;
      left: -5px;
      right: -5px;
      border: 2px solid ${f.colors.primary600};
    }
  }
`},30181:function($,c,t){"use strict";t.d(c,{O:function(){return E}});var n=t(74512),o=t(32735),s=t(8471),a=t(16833),f=t(17856);const e=()=>(0,n.jsx)(a.x,{"aria-hidden":!0,paddingLeft:1,paddingRight:1,children:(0,n.jsx)(f.Z,{variant:"pi",textColor:"neutral500",children:"/"})});e.displayName="Divider";var i=t(5717);const h=(0,s.ZP)(i.k)`
  // CrumbLinks do have padding-x, because they need to have a
  // interaction effect, which mis-aligns the breadcrumbs on the left.
  // This normalizes the behavior by moving the first item to left by
  // the same amount it has inner padding
  :first-child {
    margin-left: ${({theme:p})=>`calc(-1*${p.spaces[2]})`};
  }
`,E=({label:p,children:_,...w})=>{const b=o.Children.toArray(_);return(0,n.jsx)(a.x,{"aria-label":p,...w,children:(0,n.jsx)(h,{as:"ol",children:o.Children.map(b,(P,m)=>{const u=b.length>1&&m+1<b.length;return(0,n.jsxs)(i.k,{inline:!0,as:"li",children:[P,u&&(0,n.jsx)(e,{})]})})})})};E.displayName="Breadcrumbs"},15290:function($,c,t){"use strict";t.d(c,{$:function(){return a}});var n=t(74512),o=t(16833),s=t(17856);const a=({children:f,isCurrent:e=!1,...i})=>(0,n.jsx)(o.x,{paddingLeft:2,paddingRight:2,paddingTop:1,paddingBottom:1,children:(0,n.jsx)(s.Z,{variant:"pi",textColor:"neutral800",fontWeight:e?"bold":"normal","aria-current":e,...i,children:f})});a.displayName="Crumb"},53083:function($,c,t){"use strict";t.d(c,{VY:function(){return $n},ck:function(){return mn},fC:function(){return vn},xz:function(){return z}});var n=t(74512),o=t(32735),s=t(5904),a=t(91917),f=t(51939),e=t(8471),i=t(40187),h=t(95441),E=t(69549),p=t(55171),_=t(17856);const w=(0,e.ZP)(p.f)`
  display: inline-flex;
  align-items: center;
  text-decoration: none;
  gap: ${({theme:d})=>d.spaces[2]};
  pointer-events: ${({disabled:d})=>d?"none":void 0};

  svg {
    font-size: ${10/16}rem;

    path {
      fill: ${({disabled:d,theme:r})=>d?r.colors.neutral600:r.colors.primary600};
    }
  }

  &:hover {
    color: ${({theme:d})=>d.colors.primary500};
  }

  &:active {
    color: ${({theme:d})=>d.colors.primary700};
  }

  ${E.BF};
`,b=o.forwardRef(({children:d,href:r,disabled:l=!1,startIcon:x,endIcon:g,isExternal:v=!0,...C},D)=>(0,n.jsxs)(w,{ref:D,href:r,disabled:l,isExternal:v,...C,children:[x,(0,n.jsx)(_.Z,{textColor:l?"neutral600":"primary600",children:d}),g,r&&!g&&v&&(0,n.jsx)(h.Z,{})]}));b.displayName="Link";var P=t(75760);const m="success-light",u="danger-light",M="default",j="tertiary",y="secondary",R="danger",O="success",T="ghost",L=[m,u],W=[M,j,y,R,O,T,...L],N=["S","M","L"],A=d=>d===m||d===u?`${d.substring(0,d.lastIndexOf("-"))}`:d===j?"neutral":d===M||d===y||W.every(r=>r!==d)?"primary":`${d}`,J=({theme:d})=>`
    border: 1px solid ${d.colors.neutral200};
    background: ${d.colors.neutral150};
    ${_.Z} {
      color: ${d.colors.neutral600};
    }
    svg {
      > g, path {
        fill: ${d.colors.neutral600};
      }
    }
  `,en=({theme:d,variant:r})=>[...L,y].includes(r)?`
      background-color: ${d.colors.neutral0};
    `:r===j?`
      background-color: ${d.colors.neutral100};
    `:r===T?`
      background-color: ${d.colors.neutral100};
    `:r===M?`
      border: 1px solid ${d.colors.buttonPrimary500};
      background: ${d.colors.buttonPrimary500};
    `:`
    border: 1px solid ${d.colors[`${A(r)}500`]};
    background: ${d.colors[`${A(r)}500`]};
  `,rn=({theme:d,variant:r})=>[...L,y].includes(r)?`
      background-color: ${d.colors.neutral0};
      border: 1px solid ${d.colors[`${A(r)}600`]};
      ${_.Z} {
        color: ${d.colors[`${A(r)}600`]};
      }
      svg {
        > g, path {
          fill: ${d.colors[`${A(r)}600`]};
        }
      }
    `:r===j?`
      background-color: ${d.colors.neutral150};
    `:`
    border: 1px solid ${d.colors[`${A(r)}600`]};
    background: ${d.colors[`${A(r)}600`]};
  `,G=({theme:d,variant:r})=>{switch(r){case u:case m:case y:return`
          border: 1px solid ${d.colors[`${A(r)}200`]};
          background: ${d.colors[`${A(r)}100`]};
          ${_.Z} {
            color: ${d.colors[`${A(r)}700`]};
          }
          svg {
            > g, path {
              fill: ${d.colors[`${A(r)}700`]};
            }
          }
        `;case j:return`
          border: 1px solid ${d.colors.neutral200};
          background: ${d.colors.neutral0};
          ${_.Z} {
            color: ${d.colors.neutral800};
          }
          svg {
            > g, path {
              fill: ${d.colors.neutral800};
            }
          }
        `;case T:return`
        border: 1px solid transparent;
        background: transparent;

        ${_.Z} {
          color: ${d.colors.neutral800};
        }

        svg {
          > g, path {
            fill: ${d.colors.neutral500};
          }
        }
      `;case O:case R:return`
          border: 1px solid ${d.colors[`${A(r)}600`]};
          background: ${d.colors[`${A(r)}600`]};
          ${_.Z} {
            color: ${d.colors.neutral0};
          }
        `;default:return`
          svg {
            > g, path {
              fill: ${d.colors.buttonNeutral0};
            }
          }
        `}};var on=t(83819),q=t(16833),Y=t(5717);const sn=(0,e.F4)`
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(359deg);
  }
`,an=(0,e.ZP)(P.Z)`
  animation: ${sn} 2s infinite linear;
  will-change: transform;
`,ln=(0,e.ZP)(on.Y)`
  height: ${({theme:d,size:r})=>d.sizes.button[r]};

  svg {
    height: ${12/16}rem;
    width: auto;
  }

  &[aria-disabled='true'] {
    ${J}

    &:active {
      ${J}
    }
  }

  &:hover {
    ${en}
  }

  &:active {
    ${rn}
  }

  ${G}
`,Q=o.forwardRef(({variant:d=M,startIcon:r,endIcon:l,disabled:x=!1,children:g,onClick:v,size:C=N[0],loading:D=!1,fullWidth:B=!1,...I},Z)=>{const S=x||D,F=U=>{!S&&v&&v(U)};return(0,n.jsxs)(ln,{ref:Z,"aria-disabled":S,disabled:S,size:C,variant:d,onClick:F,fullWidth:B,alignItems:"center",background:"buttonPrimary600",borderColor:"buttonPrimary600",gap:2,inline:B,justifyContent:B?"center":void 0,paddingLeft:4,paddingRight:4,width:B?"100%":void 0,...I,children:[(r||D)&&(0,n.jsx)(q.x,{"aria-hidden":!0,children:D?(0,n.jsx)(an,{}):r}),(0,n.jsx)(_.Z,{variant:C==="S"?"pi":void 0,fontWeight:"bold",textColor:"buttonNeutral0",children:g}),l&&(0,n.jsx)(Y.k,{"aria-hidden":!0,children:l})]})});Q.displayName="Button";const dn=s.fC,cn=(0,o.forwardRef)(({size:d,endIcon:r=(0,n.jsx)(a.Z,{width:`${6/16}rem`,height:`${4/16}rem`,"aria-hidden":!0}),...l},x)=>(0,n.jsx)(s.xz,{asChild:!0,children:(0,n.jsx)(Q,{ref:x,type:"button",variant:"ghost",endIcon:r,paddingTop:d==="S"?1:2,paddingBottom:d==="S"?1:2,paddingLeft:d==="S"?3:4,paddingRight:d==="S"?3:4,...l})})),un=(0,o.forwardRef)(({children:d,intersectionId:r,popoverPlacement:l="bottom-start",...x},g)=>{const[v,C]=l.split("-");return(0,n.jsx)(s.Uv,{children:(0,n.jsx)(s.VY,{align:C,side:v,loop:!0,asChild:!0,children:(0,n.jsxs)(k,{ref:g,direction:"column",borderStyle:"solid",borderWidth:"1px",borderColor:"neutral150",hasRadius:!0,background:"neutral0",shadow:"filterShadow",maxHeight:"15rem",padding:1,alignItems:"flex-start",position:"relative",overflow:"auto",...x,children:[d,(0,n.jsx)(q.x,{id:r,width:"100%",height:"1px"})]})})})}),k=(0,e.ZP)(Y.k)`
  scrollbar-width: none;
  -ms-overflow-style: none;
  -webkit-overflow-scrolling: touch;

  &::-webkit-scrollbar {
    display: none;
  }
`,fn=({onSelect:d,disabled:r=!1,...l})=>(0,n.jsx)(s.ck,{asChild:!0,onSelect:d,disabled:r,children:l.isLink||l.isExternal?(0,n.jsx)(hn,{color:"neutral800",...l,isExternal:l.isExternal??!1,children:(0,n.jsx)(_.Z,{children:l.children})}):(0,n.jsx)(tn,{cursor:"pointer",color:"neutral800",background:"transparent",borderStyle:"none",...l,children:(0,n.jsx)(_.Z,{children:l.children})})}),nn=({theme:d})=>(0,e.iv)`
  text-align: left;
  width: 100%;
  border-radius: ${d.borderRadius};
  padding: ${d.spaces[2]} ${d.spaces[4]};

  ${_.Z} {
    color: inherit;
  }

  &[aria-disabled] {
    cursor: not-allowed;

    ${_.Z} {
      color: ${d.colors.neutral500};
    }
  }

  &[data-highlighted] {
    background-color: ${d.colors.primary100};
  }

  &:focus-visible {
    outline: none;

    &:after {
      content: none;
    }
  }
`,tn=(0,e.ZP)(Y.k)`
  ${nn}
`,hn=(0,e.ZP)(b)`
  /* We include this here again because typically when people use OptionLink they provide an as prop which cancels the Box props */
  color: ${({theme:d,color:r})=>(0,i.$)(d.colors,r,void 0)};
  text-decoration: none;

  &:hover {
    color: unset;
  }

  svg > path,
  &:focus-visible svg > path {
    fill: currentColor;
  }

  ${nn}
`,jn=(0,o.forwardRef)((d,r)=>(0,n.jsx)(s.__,{asChild:!0,children:(0,n.jsx)(pn,{ref:r,variant:"sigma",textColor:"neutral600",...d})})),pn=(0,e.ZP)(_.Z)`
  padding: ${({theme:d})=>d.spaces[2]} ${({theme:d})=>d.spaces[4]};
`,Pn=s.Tr,xn=(0,o.forwardRef)(({disabled:d=!1,...r},l)=>(0,n.jsx)(s.fF,{asChild:!0,disabled:d,children:(0,n.jsxs)(gn,{ref:l,color:"neutral800",as:"button",type:"button",background:"transparent",borderStyle:"none",gap:5,...r,children:[(0,n.jsx)(_.Z,{children:r.children}),(0,n.jsx)(_n,{height:12,width:12})]})})),gn=(0,e.ZP)(tn)`
  &[data-state='open'] {
    background-color: ${({theme:d})=>d.colors.primary100};
  }
`,_n=(0,e.ZP)(f.Z)`
  path {
    fill: ${({theme:d})=>d.colors.neutral500};
  }
`,Mn=(0,o.forwardRef)((d,r)=>(0,n.jsx)(s.Uv,{children:(0,n.jsx)(s.tu,{sideOffset:8,asChild:!0,children:(0,n.jsx)(k,{ref:r,direction:"column",borderStyle:"solid",borderWidth:"1px",borderColor:"neutral150",hasRadius:!0,background:"neutral0",shadow:"filterShadow",maxHeight:"15rem",padding:1,alignItems:"flex-start",overflow:"auto",...d})})})),vn=dn,z=cn,$n=un,mn=fn,wn=null,En=null,yn=null,bn=null},43240:function($,c,t){"use strict";t.d(c,{m:function(){return E}});var n=t(74512),o=t(8471),s=t(40041),a=t(16833);const f=(0,o.ZP)(a.x)`
  display: grid;
  grid-template-columns: repeat(${({gridCols:p})=>p}, 1fr);
  ${({theme:p,gap:_})=>(0,s.Z)("gap",_,p)}
`,e=p=>{const{gap:_="0",gridCols:w=12,...b}=p;return(0,n.jsx)(f,{gap:_,gridCols:w,...b})},i=`${232/16}rem`,h=(0,o.ZP)(e)`
  width: ${i};
  background: ${({theme:p})=>p.colors.neutral100};
  position: sticky;
  top: 0;
  height: 100vh;
  overflow-y: auto;
  border-right: 1px solid ${({theme:p})=>p.colors.neutral200};
  z-index: 1;
`,E=({ariaLabel:p,..._})=>(0,n.jsx)(h,{"aria-label":p,as:"nav",..._})},40465:function($,c,t){"use strict";t.d(c,{p:function(){return d}});var n=t(74512),o=t(32735),s=t(17229),a=t(8471);const f={DOWN:"ArrowDown",UP:"ArrowUp",RIGHT:"ArrowRight",LEFT:"ArrowLeft",ESCAPE:"Escape",ENTER:"Enter",SPACE:" ",TAB:"Tab",END:"End",HOME:"Home",DELETE:"Delete",PAGE_UP:"PageUp",PAGE_DOWN:"PageDown",BACKSPACE:"Backspace",CLEAR:"Clear"};var e=t(77062);const i=r=>{const l=(0,o.useRef)();return(0,o.useEffect)(()=>{l.current=r}),l.current};var h=t(16833);const E=(0,a.ZP)(h.x)`
  height: 1px;
  border: none;
  /* If contained in a Flex parent we want to prevent the Divider to shink */
  flex-shrink: 0;
  ${({unsetMargin:r})=>r?"margin: 0;":""}
`,p=({unsetMargin:r=!0,...l})=>(0,n.jsx)(E,{...l,background:"neutral150",as:"hr",unsetMargin:r}),_=r=>(0,n.jsx)("form",{...r,role:"search"});var w=t(98610);function b(r,l){typeof r=="function"?r(l):r!=null&&(r.current=l)}function P(...r){return l=>r.forEach(x=>b(x,l))}function m(...r){return React.useCallback(P(...r),r)}var u=t(69549);const M=(0,o.createContext)({id:"",required:!1}),j=()=>(0,o.useContext)(M);var y=t(5717);const R={S:6.5,M:10.5},O=(0,o.forwardRef)(({endAction:r,startAction:l,disabled:x=!1,onChange:g,size:v="M",...C},D)=>{const{id:B,error:I,hint:Z,name:S,required:F}=j();let U;I?U=`${B}-error`:Z&&(U=`${B}-hint`);const K=Boolean(I),H=X=>{!x&&g&&g(X)};return(0,n.jsxs)(L,{justifyContent:"space-between",hasError:K,disabled:x,children:[l?(0,n.jsx)(h.x,{paddingLeft:3,paddingRight:2,children:l}):null,(0,n.jsx)(T,{id:B,name:S,ref:D,"aria-describedby":U,"aria-invalid":K,"aria-disabled":x,disabled:x,"data-disabled":x?"":void 0,hasLeftAction:Boolean(l),hasRightAction:Boolean(r),onChange:H,"aria-required":F,$size:v,...C}),r?(0,n.jsx)(h.x,{paddingLeft:2,paddingRight:3,children:r}):null]})}),T=a.ZP.input`
  border: none;
  border-radius: ${({theme:r})=>r.borderRadius};
  padding-bottom: ${({$size:r})=>`${R[r]/16}rem`};
  padding-left: ${({theme:r,hasLeftAction:l})=>l?0:r.spaces[4]};
  padding-right: ${({theme:r,hasRightAction:l})=>l?0:r.spaces[4]};
  padding-top: ${({$size:r})=>`${R[r]/16}rem`};
  cursor: ${r=>r["aria-disabled"]?"not-allowed":void 0};

  color: ${({theme:r})=>r.colors.neutral800};
  font-weight: 400;
  font-size: ${r=>r.theme.fontSizes[2]};
  display: block;
  width: 100%;
  background: inherit;

  ::placeholder {
    color: ${({theme:r})=>r.colors.neutral500};
    opacity: 1;
  }

  &[aria-disabled='true'] {
    color: inherit;
  }

  //focus managed by InputWrapper
  &:focus {
    outline: none;
    box-shadow: none;
  }
`,L=(0,a.ZP)(y.k)`
  border: 1px solid ${({theme:r,hasError:l})=>l?r.colors.danger600:r.colors.neutral200};
  border-radius: ${({theme:r})=>r.borderRadius};
  background: ${({theme:r})=>r.colors.neutral0};
  ${(0,u.k3)()}

  ${({theme:r,disabled:l})=>l?(0,a.iv)`
          color: ${r.colors.neutral600};
          background: ${r.colors.neutral150};
        `:void 0}
`,W=a.ZP.div`
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
`,N=(0,a.ZP)(y.k)`
  font-size: 1.6rem;
  padding: 0;
`,A=(0,o.forwardRef)(({label:r,children:l,...x},g)=>(0,n.jsxs)(N,{justifyContent:"unset",background:"transparent",borderStyle:"none",type:"button",...x,as:"button",ref:g,children:[(0,n.jsx)(W,{as:"span",children:r}),(0,o.cloneElement)(l,{"aria-hidden":!0,focusable:!1})]})),J=(0,o.forwardRef)(({children:r,name:l,error:x,hint:g,id:v,required:C=!1,...D},B)=>{const I=(0,e.M)(v),Z=(0,o.useMemo)(()=>({name:l,id:I,error:x,hint:g,required:C}),[x,I,g,l,C]);return(0,n.jsx)(h.x,{ref:B,...D,children:(0,n.jsx)(M.Provider,{value:Z,children:r})})}),en="[@strapi/design-system]:",rn=r=>{const l=r;let x=!1;if(typeof l!="function")throw new TypeError(`${en} once requires a function parameter`);return(...g)=>{x||(l(...g),x=!0)}};var G=t(17856);const on=rn(console.warn),q=(0,o.forwardRef)(({children:r,action:l,required:x,...g},v)=>{const{id:C,required:D}=j(),B=D||x;return x!==void 0&&on('Deprecation warning: Usage of "required" prop in FieldLabel component is deprecated. This is discouraged and will be removed in the next major release. Please use the Field component to share the required prop.'),(0,n.jsxs)(Y,{ref:v,variant:"pi",textColor:"neutral800",htmlFor:C,fontWeight:"bold",as:"label",...g,children:[r,B&&(0,n.jsx)(sn,{textColor:"danger600",children:"*"}),l&&(0,n.jsx)(an,{marginLeft:1,children:l})]})}),Y=(0,a.ZP)(G.Z)`
  display: flex;
  align-items: center;
`,sn=(0,a.ZP)(G.Z)`
  line-height: 0;
`,an=(0,a.ZP)(y.k)`
  line-height: 0;

  svg path {
    fill: ${({theme:r})=>r.colors.neutral500};
  }
`,ln=(0,a.ZP)(w.Z)`
  font-size: 0.5rem;
  path {
    fill: ${({theme:r})=>r.colors.neutral400};
  }
`,Q=(0,a.ZP)(s.Z)`
  font-size: 0.8rem;
  path {
    fill: ${({theme:r})=>r.colors.neutral800};
  }
`,dn=a.ZP.div`
  border-radius: ${({theme:r})=>r.borderRadius};
  box-shadow: ${({theme:r})=>r.shadows.filterShadow};

  &:focus-within {
    ${Q} {
      path {
        fill: ${({theme:r})=>r.colors.primary600};
      }
    }
  }

  ${L} {
    border: 1px solid transparent;
  }

  ${(0,u.k3)(L)}
`,cn=(0,o.forwardRef)(({name:r,size:l="M",children:x,value:g="",onClear:v,clearLabel:C,...D},B)=>{const I=(0,o.useRef)(null),Z=g.length>0,S=U=>{v(U),I.current.focus()},F=P(B,I);return(0,n.jsx)(dn,{children:(0,n.jsxs)(J,{name:r,children:[(0,n.jsx)(W,{children:(0,n.jsx)(q,{children:x})}),(0,n.jsx)(O,{ref:F,value:g,startAction:(0,n.jsx)(Q,{"aria-hidden":!0}),size:l,endAction:Z?(0,n.jsx)(A,{label:C,onClick:S,children:(0,n.jsx)(ln,{})}):void 0,...D})]})})}),un=r=>{const[l,x]=(0,o.useState)(!1),g=(0,o.useRef)(null),v=()=>{typeof g.current=="number"&&(clearTimeout(g.current),g.current=null)};return(0,o.useEffect)(()=>()=>{v()},[]),{visible:l,onFocus:()=>{x(!0)},onBlur:()=>{x(!1)},onMouseEnter:()=>{g.current=setTimeout(()=>{x(!0)},r)},onMouseLeave:()=>{v(),x(!1)}}},k=8,fn=(r,l)=>{const x=(r.width-l.width)/2,g=l.left-x,v=l.top+l.height+k+window.pageYOffset;return{left:g,top:v}},nn=(r,l)=>{const x=(r.height-l.height)/2,g=l.left+l.width+k,v=l.top-x+window.pageYOffset;return{left:g,top:v}},tn=(r,l)=>{const x=(r.height-l.height)/2,g=l.left-r.width-k,v=l.top-x+window.pageYOffset;return{left:g,top:v}},hn=(r,l)=>{const x=(r.width-l.width)/2;let g=l.left-x,v=l.top-r.height-k+window.pageYOffset;const C=window.innerWidth-l.right;return l.left+r.width-C>window.innerWidth?(g=l.left-r.width-k,v=l.top+window.scrollY-l.height/2):g<0?(g=l.width+l.left+k,v=l.top+window.scrollY-r.height/2+k):v<0&&g>0&&(v=l.top+l.height+k),{left:g,top:v}},jn=(r,l,x)=>{const g=r.getBoundingClientRect(),v=l.getBoundingClientRect();return x==="bottom"?fn(g,v):x==="right"?nn(g,v):x==="left"?tn(g,v):hn(g,v)},pn=(r,l)=>{const x=(0,o.useRef)(null),g=(0,o.useRef)(null);return(0,o.useLayoutEffect)(()=>{if(r){const v=x.current,C=g.current;if(v&&C){const D=jn(v,C,l);v.style.left=`${D.left}px`,v.style.top=`${D.top}px`}}},[l,r]),{tooltipWrapperRef:x,toggleSourceRef:g}};var Pn=t(12788);const xn=o.forwardRef(({container:r=globalThis?.document?.body,...l},x)=>r?(0,Pn.createPortal)((0,n.jsx)(h.x,{ref:x,...l}),r):null);xn.displayName="Portal";const gn=(0,a.ZP)(h.x)`
  /* z-index exist because of its position inside Modals */
  z-index: 4;
  display: ${({visible:r})=>r?"revert":"none"};
`,_n=({children:r,label:l,description:x,delay:g=500,position:v="top",id:C,...D})=>{const B=(0,e.M)(C),I=(0,e.M)(),{visible:Z,...S}=un(g),{tooltipWrapperRef:F,toggleSourceRef:U}=pn(Z,v),K=o.cloneElement(r,{tabIndex:0,"aria-labelledby":l?B:void 0,"aria-describedby":x?B:void 0,...S});return(0,n.jsxs)(n.Fragment,{children:[(0,n.jsx)(xn,{children:(0,n.jsxs)(gn,{id:B,background:"neutral900",hasRadius:!0,padding:2,role:"tooltip",ref:F,visible:Z,position:"absolute",...D,children:[Z&&(0,n.jsx)(W,{id:I,children:x}),(0,n.jsx)(G.Z,{as:"p",variant:"pi",fontWeight:"bold",textColor:"neutral0",children:l||x})]})}),(0,n.jsx)("span",{ref:U,children:K})]})};var Mn=t(83819);const vn="tertiary",z="secondary",$n=["S","M","L"],mn=[vn,z],wn=o.forwardRef(({label:r,background:l,borderWidth:x,noBorder:g=!1,children:v,icon:C,disabled:D=!1,onClick:B,size:I=$n[0],"aria-label":Z,variant:S=mn[0],...F},U)=>{const K=X=>{!D&&B&&B(X)},H=(0,n.jsxs)(En,{"aria-disabled":D,background:D?"neutral150":l,borderWidth:g?0:x,justifyContent:"center",...F,ref:U,size:I,onClick:K,variant:S,children:[(0,n.jsx)(W,{as:"span",children:r??Z}),o.cloneElement(C||v,{"aria-hidden":!0,focusable:!1})]});return r?(0,n.jsx)(_n,{label:r,children:H}):H}),En=(0,a.ZP)(Mn.Y)`
  background-color: ${({theme:r,variant:l})=>{if(l===z)return r.colors.primary100}};
  border-color: ${({theme:r,variant:l})=>l===z?r.colors.primary200:r.colors.neutral200};
  height: ${({theme:r,size:l})=>r.sizes.button[l]};
  width: ${({theme:r,size:l})=>r.sizes.button[l]};

  svg {
    g,
    path {
      fill: ${({theme:r,variant:l})=>l===z?r.colors.primary500:r.colors.neutral500};
    }
  }

  :hover,
  :focus {
    svg {
      g,
      path {
        fill: ${({theme:r,variant:l})=>l===z?r.colors.primary600:r.colors.neutral600};
      }
    }
  }

  &[aria-disabled='true'] {
    svg {
      path {
        fill: ${({theme:r})=>r.colors.neutral600};
      }
    }
  }
`,yn=(0,a.ZP)(y.k)`
  & span:first-child button {
    border-left: 1px solid ${({theme:r})=>r.colors.neutral200};
    border-radius: ${({theme:r})=>`${r.borderRadius} 0 0 ${r.borderRadius}`};
  }

  & span:last-child button {
    border-radius: ${({theme:r})=>`0 ${r.borderRadius} ${r.borderRadius} 0`};
  }

  & ${En} {
    border-radius: 0;
    border-left: none;

    svg {
      path {
        fill: ${({theme:r})=>r.colors.neutral700};
      }
    }

    &:hover {
      background-color: ${({theme:r})=>r.colors.neutral100};

      svg {
        path {
          fill: ${({theme:r})=>r.colors.neutral800};
        }
      }
    }

    &:active {
      background-color: ${({theme:r})=>r.colors.neutral150};
      svg {
        path {
          fill: ${({theme:r})=>r.colors.neutral900};
        }
      }
    }

    &[aria-disabled='true'] {
      svg {
        path {
          fill: ${({theme:r})=>r.colors.neutral600};
        }
      }
    }
  }
`,bn=(0,a.ZP)(p)`
  width: ${24/16}rem;
  background-color: ${({theme:r})=>r.colors.neutral200};
`,d=({as:r="h2",label:l,searchLabel:x="",searchable:g=!1,onChange:v=()=>{},value:C="",onClear:D=()=>{},onSubmit:B=()=>{},id:I})=>{const[Z,S]=(0,o.useState)(!1),F=i(Z),U=(0,e.M)(I),K=(0,o.useRef)(void 0),H=(0,o.useRef)(void 0);(0,o.useEffect)(()=>{Z&&K.current&&K.current.focus(),F&&!Z&&H.current&&H.current.focus()},[Z,F]);const X=()=>{S(V=>!V)},Cn=V=>{D(V),K.current.focus()},Dn=V=>{V.relatedTarget?.id!==U&&S(!1)},Bn=V=>{V.key===f.ESCAPE&&S(!1)};return Z?(0,n.jsxs)(h.x,{paddingLeft:4,paddingTop:5,paddingBottom:2,paddingRight:4,children:[(0,n.jsx)(_,{children:(0,n.jsx)(cn,{name:"searchbar",value:C,onChange:v,placeholder:"e.g: strapi-plugin-abcd",onKeyDown:Bn,ref:K,onBlur:Dn,onClear:Cn,onSubmit:B,clearLabel:"Clear",size:"S",children:x})}),(0,n.jsx)(h.x,{paddingLeft:2,paddingTop:4,children:(0,n.jsx)(bn,{})})]}):(0,n.jsxs)(h.x,{paddingLeft:6,paddingTop:6,paddingBottom:2,paddingRight:4,children:[(0,n.jsxs)(y.k,{justifyContent:"space-between",alignItems:"flex-start",children:[(0,n.jsx)(G.Z,{variant:"beta",as:r,children:l}),g&&(0,n.jsx)(wn,{ref:H,onClick:X,label:x,icon:(0,n.jsx)(s.Z,{})})]}),(0,n.jsx)(h.x,{paddingTop:4,children:(0,n.jsx)(bn,{})})]})}},49211:function($,c,t){"use strict";t.d(c,{E:function(){return w}});var n=t(74512),o=t(32735),s=t(48715),a=t(8471),f=t(16833),e=t(17856),i=t(5717),h=t(55171);const E=(0,a.ZP)(f.x)`
  display: flex;
  align-items: center;
  justify-content: space-between;
  text-decoration: none;
  color: ${({theme:b})=>b.colors.neutral800};
  svg > * {
    fill: ${({theme:b})=>b.colors.neutral600};
  }

  &.active {
    ${({theme:b})=>`
      background-color: ${b.colors.primary100};
      border-right: 2px solid ${b.colors.primary600};
      svg > * {
        fill: ${b.colors.primary700};
      }
      ${e.Z} {
        color: ${b.colors.primary700};
        font-weight: 500;
      }
      `}
  }

  &:focus-visible {
    outline-offset: -2px;
  }
`,p=(0,a.ZP)(s.Z)`
  width: ${12/16}rem;
  height: ${4/16}rem;
  * {
    fill: ${({theme:b,$active:P})=>P?b.colors.primary600:b.colors.neutral600};
  }
`,_=a.ZP.div`
  svg {
    height: ${12/16}rem;
    width: ${12/16}rem;
  }
`,w=o.forwardRef(({children:b,icon:P=null,withBullet:m=!1,as:u=h.f,isSubSectionChild:M=!1,...j},y)=>(0,n.jsxs)(E,{as:u,icon:P,background:"neutral100",paddingLeft:M?9:7,paddingBottom:2,paddingTop:2,ref:y,...j,children:[(0,n.jsxs)(i.k,{children:[P?(0,n.jsx)(_,{children:P}):(0,n.jsx)(p,{}),(0,n.jsx)(f.x,{paddingLeft:2,children:(0,n.jsx)(e.Z,{as:"span",children:b})})]}),m&&(0,n.jsx)(f.x,{as:i.k,paddingRight:4,children:(0,n.jsx)(p,{$active:!0})})]}))},97614:function($,c,t){"use strict";t.d(c,{z:function(){return w}});var n=t(74512),o=t(32735),s=t(91917),a=t(8471),f=t(77062),e=t(16833),i=t(5717),h=t(17856);const E=(0,a.ZP)(e.x)`
  svg {
    height: ${4/16}rem;
    path {
      fill: ${({theme:b})=>b.colors.neutral700};
    }
  }
`,p=a.ZP.button`
  border: none;
  padding: 0;
  background: transparent;
  display: flex;
  align-items: center;
`,_=a.ZP.div`
  display: flex;
  align-items: center;
  width: ${12/16}rem;
  transform: rotateX(${({rotated:b})=>b?"0deg":"180deg"});
`,w=({label:b,children:P,id:m})=>{const[u,M]=(0,o.useState)(!0),j=(0,f.M)(m),y=()=>{M(R=>!R)};return(0,n.jsxs)(e.x,{children:[(0,n.jsx)(E,{paddingLeft:7,paddingTop:2,paddingBottom:2,paddingRight:4,children:(0,n.jsx)(i.k,{justifyContent:"space-between",children:(0,n.jsxs)(p,{onClick:y,"aria-expanded":u,"aria-controls":j,children:[(0,n.jsx)(_,{rotated:u,children:(0,n.jsx)(s.Z,{"aria-hidden":!0})}),(0,n.jsx)(e.x,{paddingLeft:2,children:(0,n.jsx)(h.Z,{as:"span",fontWeight:"semiBold",textColor:"neutral800",children:b})})]})})}),u&&(0,n.jsx)("ul",{id:j,children:o.Children.map(P,(R,O)=>(0,n.jsx)("li",{children:R},O))})]})}},24164:function($,c,t){"use strict";t.d(c,{D:function(){return m}});var n=t(74512),o=t(32735),s=t(8471),a=t(91917),f=t(5717),e=t(16833),i=t(17856);const h=(0,s.ZP)(f.k)`
  border: none;
  padding: 0;
  background: transparent;
`,E=s.ZP.div`
  display: flex;
  align-items: center;
  transform: rotateX(${({rotated:u})=>u?"0deg":"180deg"});
`,p=({collapsable:u=!1,label:M,onClick:j=()=>{},ariaExpanded:y,ariaControls:R})=>u?(0,n.jsxs)(h,{as:"button",onClick:j,"aria-expanded":y,"aria-controls":R,textAlign:"left",children:[(0,n.jsx)(e.x,{paddingRight:1,children:(0,n.jsx)(i.Z,{variant:"sigma",textColor:"neutral600",children:M})}),u&&(0,n.jsx)(E,{rotated:y,children:(0,n.jsx)(a.Z,{"aria-hidden":!0})})]}):(0,n.jsx)(h,{children:(0,n.jsx)(e.x,{paddingRight:1,children:(0,n.jsx)(i.Z,{variant:"sigma",textColor:"neutral600",children:M})})});var _=t(77062);const w=(0,s.ZP)(f.k)`
  border-radius: ${({theme:u,size:M})=>M==="S"?"2px":u.borderRadius};
  height: ${({size:u,theme:M})=>M.sizes.badge[u]};
`,b=({active:u=!1,size:M="M",textColor:j="neutral600",backgroundColor:y="neutral150",children:R,minWidth:O=5,...T})=>{const L=M==="S"?1:2;return(0,n.jsx)(w,{inline:!0,alignItem:"center",justifyContent:"center",minWidth:O,paddingLeft:L,paddingRight:L,background:u?"primary200":y,size:M,...T,children:(0,n.jsx)(i.Z,{variant:"sigma",textColor:u?"primary600":j,children:R})})},P=(0,s.ZP)(e.x)`
  svg {
    height: ${4/16}rem;
    path {
      fill: ${({theme:u})=>u.colors.neutral500};
    }
  }
`,m=({collapsable:u=!1,label:M,badgeLabel:j,children:y,id:R})=>{const[O,T]=(0,o.useState)(!0),L=(0,_.M)(R),W=()=>{T(N=>!N)};return(0,n.jsxs)(f.k,{direction:"column",alignItems:"stretch",gap:1,children:[(0,n.jsx)(P,{paddingLeft:6,paddingTop:2,paddingBottom:2,paddingRight:4,children:(0,n.jsxs)(e.x,{position:"relative",paddingRight:j?6:0,children:[(0,n.jsx)(p,{onClick:W,ariaExpanded:O,ariaControls:L,collapsable:u,label:M}),j&&(0,n.jsx)(b,{backgroundColor:"neutral150",textColor:"neutral600",position:"absolute",right:0,top:"50%",transform:"translateY(-50%)",children:j})]})}),(!u||O)&&(0,n.jsx)("ol",{id:L,children:o.Children.map(y,(N,A)=>(0,n.jsx)("li",{children:N},A))})]})}},31327:function($,c,t){"use strict";t.d(c,{Z:function(){return f}});var n=t(74512),o=t(32735),s=t(16833),a=t(5717);const f=({children:e,spacing:i=2,horizontal:h=!1,...E})=>(0,n.jsx)(s.x,{paddingTop:2,paddingBottom:4,children:(0,n.jsx)(a.k,{as:"ol",gap:i,direction:h?"row":"column",alignItems:h?"center":"stretch",...E,children:o.Children.map(e,(p,_)=>(0,n.jsx)("li",{children:p},_))})})}}]);
